apps/condo/lang/en/en.json

```diff
@@ -2196,6 +2196,8 @@
   "api.billing.sendNewReceiptMessagesToResidentScopes.INVALID_BILLING_CATEGORY_PROVIDED": "Please provide existing non-deleted billing category id",
   "api.billing.sendNewReceiptMessagesToResidentScopes.SCOPES_IS_EMPTY": "Scopes could not be empty",
   "api.billing.sendNewReceiptMessagesToResidentScopes.INVALID_BILLING_PROPERTY_PROVIDED": "All provided billing properties in scopes should be non-deleted & connected to organization of the billing context",
+  "api.billing.registerBillingReceiptFile.MULTIPLE_BILLING_RECEIPTS_FOUND": "There is more then one receipt for this receipt file",
+  "api.billing.registerBillingReceiptFile.NO_RECEIPTS_FOUND": "There is no receipt for this receipt file",
```

Comment:
better to specifically describe which model is not found. BillingReceipt not found

Reply:
Donw